src/ReviewPanel.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react'
+import { Panel, Row, Col } from 'react-bootstrap'
+
+export default class ReviewPanel extends Component {
```

Comment:
Yes, [TeamMemberDisplay](https://github.com/maryvilledev/skilldirectoryui/blob/master/src/TeamMemberDisplay.jsx) is an example, as well as [DeleteModal](https://github.com/maryvilledev/skilldirectoryui/blob/master/src/DeleteModal.jsx) and [SelectedItem](https://github.com/maryvilledev/skilldirectoryui/blob/master/src/SelectedItem.jsx)